common/buildcraft/factory/client/render/RenderMiningWell.java

```diff
@@ -19,7 +19,7 @@
     private static final int COLOUR_STATUS_DONE = 0xFF_1f_10_1b; // black-ish
 
     private static final int BLOCK_LIGHT_STATUS_OFF = 0x0;
-    private static final int BLOCK_LIGHT_STATUS_TODO = 0xF;
+    private static final int BLOCK_LIGHT_STATUS_TODO = 0x1;
```

Comment:
Woops, I was calling "lightf" rather than "lighti". That explains why it wasn't working.